### DISCUSSION TITLE
Is there a way to prefer implicit assertions similar to prefer-explicit-assert

### DISCUSSION THREAD

**Discussion by mfrr1118** [testing-library/eslint-plugin-testing-library#721] [Q&A] [answered]
`prefer-explicit-assert` encourages the use of `toBeInTheDocument`, is there a way to discourage it?

So that
```js
expect(getByText("Test Table")).toBeInTheDocument();
```

Is incorrect, and 
```js
getByText("Test Table");
```

Is correct

**Comment by Belco90** [comment, marked as answer]
Hi @mfrr1118! I'm afraid we don't have a rule for such behavior.

**Reply by mfrr1118** [reply]
Gotcha. Thanks for the quick reply!
